crates/nu-std/std/mod.nu

```diff
@@ -80,7 +80,6 @@ export def --env "path add" [
         $env
             | get $path_name
             | split row (char esep)
-            | path expand
```

Comment:
why wouldn't this be `path expand --no-symlink` also? it seems like there's an opportunity to have some items in the path that aren't expanded and therefore nushell wouldn't be able to find some executables in those folders.

Reply:
Of the items in this PR, I'm least concerned about this one and wouldn't let it hold up a merge.

But, of course, I still believe `path add` should only add.

Reply:
I tend to agree that most times we should leave decisions up to the user, otherwise there's too much magic. I'm fine with leaving this the way it is now. Just need to make the CI happy. Thanks.

Reply:
Roger that. I'm going to sleep now, but I'll work on it later today. Thanks so much for taking a looking and providing thoughtful feedback.